guides/common/modules/proc_running-ansible-roles-in-check-mode.adoc

```diff
@@ -0,0 +1,19 @@
+[id="running-ansible-roles-in-check-mode_{context}"]
+
+= Running Ansible Roles in Check Mode
+
+You can run Ansible roles in check mode through the {ProjectWebUI}.
+
+.Prerequisites
+
+* You must configure your deployment to run Ansible roles.
```

Comment:
What do you mean by deployment? "Managed hosts"? Please reword if possible.

Reply:
I'm using the same wording as in 2.2, 2.3 and 2.4. Should I also change these?

Reply:
I don't think it's critical to backport this. If so, then it'd be a separate PR against 2.2, 2.3, and 2.4 but I doubt that it's necessary.

Reply:
In that case I'll leave it consistent with other sections. Any changes necessary can be made at GA.